.pre-commit-config.yaml

```diff
@@ -0,0 +1,53 @@
+minimum_pre_commit_version: "1.15"
+repos:
+  # general stuff
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      # check file system problems
+      - id: check-case-conflict
+      - id: check-symlinks
+      - id: destroyed-symlinks
+
+      # unify whitespace and line ending
+      - id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
+      - id: mixed-line-ending
+      - id: end-of-file-fixer
+
+      # sort requirements.txt files
+      - id: requirements-txt-fixer
+
+  # format python files
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+      - id: black
+        files: ^src/vorta/
```

Comment:
Shall we add formatting of tests here, as you pointed out? Also in Pyproject.toml. 🙏